packages/react/src/index.ts

```diff
@@ -5,6 +5,8 @@ export { ErrorBoundaryGroup, withErrorBoundaryGroup, useErrorBoundaryGroup } fro
 export { AsyncBoundary, withAsyncBoundary } from './AsyncBoundary'
 export { Delay, withDelay } from './Delay'
 
+export { wrap } from './wrap'
```

Comment:
> I want to deprecate all hocs

I agree about this `BREAKING CHANGES`
but if this function is cover our whole hocs, what about better naming rather than `wrap`.

My suggestion is `with`

It is like jest description, using `it` 

```
it('should error when ....'
```

it is readable continually by combining `it` + `should`

### before

```tsx
withErrBoundary(ErrorBoundary, {})(MyComponent)

// wrap case
wrap(ErrorBoundary, {})(MyComponent)
```


### after

```tsx
with(ErrorBoundary, {})(MyComponent)
```

Reply:
I considered `with` before too. but in strict mode of JavaScript, `with` is keyword.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/with

And also, because this `wrap` function is a function of a more abstract layer, it is named `wrap`, which means it wraps more clearly than `with`. Just as `connect` in react-redux has the meaning of connecting a store and a component. `with` is impossible because it is a keyword, and is there a better naming than `wrap`?

Reply:
I see, I don't know about with is keyword in strict mode. wrap is better than I thought another naming. Let's Go !